### ISSUE TITLE
Set max moves

### ISSUE THREAD

**Issue by smellymoo** [GunshipPenguin/open_flood#51]
In one way, it's really good that it's hard to solve, in another, I kinda would like a slider for how many moves to solve. Or at least a slider for easy/hard/advanced.

**Comment by bashrocks** [comment]
Yes, I'd personally like a version that removes the limit altogether in favor of a "personal best" in moves and/or time taken
